.circleci/config.yml

```diff
@@ -29,22 +28,24 @@ jobs:
             echo "moving the benefit-finder-app module into the usagov-2021 directory"
             mv usagov_benefit_finder usagov-2021/web/modules/custom
             mv html.html.twig usagov-2021/web/themes/custom/usagov/templates/html.html.twig
+            mv bin/cloudgov/entrypoint.sh usagov-2021/scripts/entrypoint.sh
 
       - run: 
           name: Pre-docker-build configurations
           command: |
             cd px-benefit-finder/usagov-2021
             mv .docker/Dockerfile-cms .
-            sed -i 's/80/8080/g' Dockerfile-cms  
+            sed -i 's/80/8080/g' Dockerfile-cms
+            sed -i 's/ENTRYPOINT/CMD/' Dockerfile-cms
             sed -i 's/80/8080/g' .docker/src-cms/etc/nginx/partials/cms.conf.tmpl
             sed -i 's/80/8080/g' .docker/src-cms/etc/nginx/partials/www.conf.tmpl
             sed -i "s/\$service\['name'\] === 'database'/stristr(\$service\['name'\], 'mysql')/"  web/sites/default/settings.php
             sed -i "s/\$settings\['hash_salt'\] = \$service\['credentials'\]\['HASH_SALT'\]/\$settings\['hash_salt'\] = \$service\['credentials'\]\['hash_salt'\]/"  web/sites/default/settings.php
             sed -i "s/\$service\['name'\] === 'storage'/stristr(\$service\['name'\], 'storage')/"  web/sites/default/settings.php
             sed -i "s/\$service\['name'\] === 'secrets'/stristr(\$service\['name'\], 'secrets')/"  web/sites/default/settings.php
-            sed -i "s/      \$settings\['s3fs.use_s3_for_public'\] = TRUE;/      \/\/      \$settings\['s3fs.use_s3_for_public'\] = TRUE;/" web/sites/default/settings.php
```

Comment:
can you explain a bit about why we are removing these settings?

Reply:
@scottqueen-bixal  We were commenting out the `s3fs.use_s3_for_public` usage in the `settings.php` file to be able to use the local drupal file system. Now that, Gene has configured the app to use s3fs, we no longer need to comment this line out. The default configuration is good for our use case.

Reply:
It is no longer needed.
USAGov settings set to use S3 as public file system.
We commented out the S3 for public setting to use Drupal public file system.
Now we restore the USAGov setting to use S3 as public file system.